.travis.yml

```diff
@@ -5,6 +5,10 @@ env:
 
 install:
   - nix-env -iA nixopsUnstable -f '<nixpkgs>'
+  - echo "binary-caches = https://cache.nixos.org https://hydra.iohk.io" > ~/nix.conf
+  - echo "binary-cache-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=" > ~/nix.conf
+  - export NIX_CONF_DIR=~
```

Comment:
This is a bad idea since the default file does bring some sanity things.

Try to workaround the 14.04 bug by doing `- export PYTHONPATH=`